lib/jekyll/webpack.rb

```diff
@@ -10,7 +10,11 @@ class Error < StandardError; end
 
     def self.build(site)
       site_dest = site.dest
-
+      # only run webpack once in _site/, not in subdirectories
+      if not site_dest.end_with?("_site")                                                                                                                                                             
```

Comment:
We want a key under the `webpack` namespace in `_config.yml` to make this optional:

``` yml
webpack:
    only_in: _site
```

Then you can have logic on this line like:

``` ruby
only_in = site.config.dig('webpack', 'only_in')
if only_in && !site_dest.end_with?(only_in)
  return
end
```

WDYT? :)

Reply:
looks perfect to me. Thank you! 

I'm not sure if I need to do anything. Github doesn't let me "approve" the change so I marked this conversation as "resolved". Please notify me if I broke anything or if further action is needed :)